libcomposefs/meson.build

```diff
@@ -1,3 +1,16 @@
+internal_source_files = files([
+  'lcfs-utils.h',
+  'lcfs-utils.c',
+])
+
+libcomposefs_internal = static_library('composefs-internal',
+  internal_source_files,
+  c_args : composefs_hash_cflags + hidden_visibility_cflags,
+  dependencies : libcrypto_dep,
```

Comment:
Does it really need the crypto libs?

Reply:
Not right now, it easily might in the future...but anyways for now I dropped it.